charts/turing/templates/turing-service.yaml

```diff
@@ -5,13 +5,7 @@ metadata:
   name: {{ include "common.set-value" (list (include "turing.fullname" .) $globServiceName) }}
   namespace: {{ .Release.Namespace }}
   labels:
-    app: {{ template "turing.fullname" . }}
-    chart: {{ template "turing.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-{{ if .Values.deployment.labels -}}
-{{ toYaml .Values.deployment.labels | indent 4 -}}
-{{- end }}
+    {{- include "turing.labels" . | nindent 4 }}
```

Comment:
This `turing.labels` helper template is used extensively by the other resources in the Turing Helm chart, except in this resource; replacing the manually defined labels here with the ones from the helper template.